src/wiki/plugins/notifications/views.py

```diff
@@ -40,14 +40,12 @@ def get_article_subscriptions(self, nyt_settings):
         ).distinct()
 
     def get_form_kwargs(self):
-        kwargs = super().get_form_kwargs(self)
+        kwargs = super().get_form_kwargs()
         kwargs['user'] = self.request.user
         return kwargs
 
     def get_context_data(self, **kwargs):
-        context = super().get_context_data(self, **kwargs)
-        if 'form' not in context:
-            context['form'] = self.get_form()
+        context = super().get_context_data(**kwargs)
```

Comment:
Did this fix a syntax error?

Reply:
No, I'll take it. It's dead code, but since I have a WIP PR #979 already open to add some tests for the notifications plugin, that change could go there instead.

Reply:
:+1: